### DISCUSSION TITLE
Possibility to filter frames with issues

### DISCUSSION THREAD

**Discussion by Josimas** [cvat-ai/cvat#3568] [Q&A] [answered]
Hi,

I can't find how to filter frame(s) (in a job) with issues?

Flow:
- After annotation reviewer opens issues.
- But how annotator can find them easily?

At this moment, we only see that it should go frame by frame and look at the Issues section.
![image](https://user-images.githubusercontent.com/43665171/130014553-3b82202a-a90b-48a5-9672-a41b7c8f5d8d.png)

I also tried to find filter > unsuccessfully.

Thanks for the help.

**Comment by bsekachev** [comment, marked as answer]
@Josimas 

From the documentation:
![image](https://user-images.githubusercontent.com/40690378/130052155-6da402f7-b72e-43e0-91ed-dc72d1dbd8cf.png)

Does it help you with your case?

**Reply by Josimas** [reply]
Thanks!
